.github/workflows/main.yml

```diff
@@ -61,7 +72,6 @@ jobs:
         target:
           - x86_64-unknown-netbsd
           - x86_64-unknown-freebsd
-          - x86_64-apple-darwin
```

Comment:
@0xpr03 Removed x86_64-apple-darwin as we've already tested code with macOS on the above `ci` workflow. I think it should be fine but would like to mention it, just in case.